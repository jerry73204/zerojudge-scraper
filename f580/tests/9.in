13 89
8 3
7 8
10 -1
6 11
10 1
12 8
4 6
4 -2
6 3
12 7
11 5
2 9
12 6
4 11
10 11
3 -2
1 3
7 5
11 10
6 9
11 -1
12 5
13 -2
9 2
3 11
1 7
5 11
3 9
12 11
9 11
6 2
5 -2
1 8
7 8
10 -2
6 11
11 10
9 11
12 8
5 -2
1 12
1 -2
6 3
2 6
3 4
5 -2
7 13
10 -2
2 12
4 -2
7 5
3 -2
10 7
2 -1
1 3
6 -1
3 4
1 -2
5 -2
1 5
8 -2
3 8
11 -1
8 1
9 11
11 7
4 6
7 8
1 -2
9 10
5 11
3 11
3 2
5 -1
9 -2
13 11
8 7
6 8
4 -2
3 1
12 3
7 -1
12 2
1 6
9 6
7 11
6 12
11 12
10 9
