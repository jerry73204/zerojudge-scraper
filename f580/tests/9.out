6 2 1 3 5 6 1 6 3 3 3 5 5